module2/lessons/intro_to_class_methods.md

```diff
@@ -0,0 +1,155 @@
+---
```

Comment:
It's nice to have this history of this lesson -- I think it's just an older version. Fine to keep in this new repo I suppose